R/aaa_values.R

```diff
@@ -141,6 +144,14 @@ value_seq <- function(object, n, original = TRUE) {
 
 value_seq_dbl <- function(object, n, original = TRUE) {
   if (n == 1 && (!is.null(object$default) & !is_unknown(object$default))) {
+    lifecycle::deprecate_warn(
+      when = "1.0.0",
+      what = "new_quant_param(default)",
+      details = c(
+        i = "This uses the `default` value of the parameter.",
+        i = "See the Details section of `?value_seq`."
+      )
+    )
     res <- object$default
```

Comment:
I think we decided that we are going to remove this branch entirely. The deprecation that comes out of this PR will mean that we no longer respect the `default` argument at all. We will just throw a warning if it is used in a constructor, and state that it is no longer going to be used.